kernel/interrupt_controller/Cargo.toml

```diff
@@ -1,21 +1,27 @@
 [package]
-authors = ["Nathan Royer <nathan.royer.pro@gmail.com>"]
-description = "Cross-platform abstraction over interrupt controllers"
+name = "interrupt_controller"
 version = "0.1.0"
+authors = [
+    "Nathan Royer <nathan.royer.pro@gmail.com>",
+    "Kevin Boos <kevinaboos@gmail.com>",
+]
+description = "Cross-platform abstraction over interrupt controllers"
 edition = "2021"
-name = "interrupt_controller"
 
 [dependencies]
 log = "0.4.8"
 cpu = { path = "../cpu" }
+memory = { path = "../memory" }
+spin = "0.9.4"
+sync_irq = { path = "../../libs/sync_irq" }
 
 [target.'cfg(target_arch = "aarch64")'.dependencies]
-sync_irq = { path = "../../libs/sync_irq" }
 arm_boards = { path = "../arm_boards" }
 memory = { path = "../memory" }
```

Comment:
This aarch64-specific memory dep needs to be removed

Reply:
thanks!